README.md

```diff
@@ -1,118 +1,7 @@
-## Eclipse Che documentation
+# Eclipse Che documentation project
 
-Che docs use Jekyll to convert `.adoc` (AsciiDoc) files into HTML pages. Docs are published at [https://www.eclipse.org/che/docs](https://www.eclipse.org/che/docs/). Updates are synced with a release cycle.
+This repository contains the documentation for the Eclipse Che. See your [Contribution guide](CONTRIBUTION.md) to learn how to contribute. We love pull requests and appreciate contributions that make docs more helpful for users.
```

Comment:
"The" is not needed before "Eclipse Che".

"See the Contribution guide". Personally, though, I'd rewrite this sentence, maybe something like "to contribute to the documentation, see the Contribution guide."